### DISCUSSION TITLE
Disable italic font in comments

### DISCUSSION THREAD

**Discussion by pureby** [mark-wiemer/ahkpp#351] [Q&A] [answered]
Hello everyone,

Is there a way to disable _italic font_ in comments for this language? I like comments greyed-out like they are now, but I don't like them italic in AHK scripts.

Thanks in advance!

**Comment by fade2gray** [comment]
Curious to see a screenshot. My comments use the same font style as the rest of the code, and they're green, not greyed-out. Maybe you changed a setting somewhere?

**Reply by pureby** [reply]
My bad, I forgot to mention I'm using Solarized Light theme. It displays comments as italic by default for all languages. 

The thing is, I like italic comments for other languages. Only for AHK, italic text interferes with how my main AHK script is split in different sections visually. So I would like to override comment style only for ahkpp.

**Comment by mark-wiemer** [comment, marked as answer]
Heyo, checking in on old issues. Unfortunately this is outside the scope of this extension. The extension "tokenizes" the code and says "this part is comments, this is a variable, etc", that's the only info it provides. The VS Code theme says "oh, this token is a comment? Let me italicize it", and that theme is entirely separate from this repo.

If you haven't found a workaround, I'd suggest just changing themes for your AHK workspaces. Unfortunately this isn't a language-specific feature and must apply to the entire workspace.

![image](https://github.com/mark-wiemer-org/ahkpp/assets/7833360/a3cb7e3f-f71f-47bb-bcc1-4d2efb18b69c)
